zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -75,6 +94,15 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
         return image
     }()
     
+    private lazy var textField : UITextField = {
+        let text = UITextField()
+        text.placeholder = "newDocumentPhotos.description.placeholder".localized
+        text.adjustsFontSizeToFitWidth = true
```

Comment:
I'm not sure this is a valid solution here. Make the placeholder text shorter instead.